packages/react-component-library/src/components/AutocompleteE/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { IconAgriculture, IconAnchor, IconBrightnessAuto, IconRemove } from '@defencedigital/icon-library'
+import { Story, Meta } from '@storybook/react'
+
+import { AutocompleteE, AutocompleteEOption, AutocompleteEProps } from './index'
+
+export default {
+  component: AutocompleteE,
+  title: 'AutocompleteE (Experimental)',
```

Comment:
Would drop the E suffix in the title here:

```suggestion
  title: 'Autocomplete (Experimental)',
```